UnitTests/ReviewServiceTests.cs

```diff
@@ -0,0 +1,195 @@
+﻿using ConestogaInsidersClub.Data;
+using ConestogaInsidersClub.Data.DataAccess;
+using ConestogaInsidersClub.Data.Models;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ServiceTests
+{
+    class ReviewServiceTests : TestBase
+    {
+        private Game testGame;
+        private ApplicationUser testUser;
+        private ApplicationUser testUser2;
+        private Review testReview;
+
+        [OneTimeSetUp]
+        public async Task BeforeAll()
+        {
+
+            using var context = new ApplicationDbContext(ContextOptions);
+
+            testGame = new Game()
+            {
+                GameId = 1,
+                Name = "Age of Empires IV",
+                Description = "RTS Game"
+            };
+            await context.Set<Game>().AddAsync(testGame);
+
+            testUser = new ApplicationUser()
+            {
+                Id = "d123",
+                UserName = "test",
+                NormalizedUserName = "TEST",
+                Email = "test@user.com"
+            };
+            testUser2 = new ApplicationUser()
+            {
+                Id = "d124",
+                UserName = "fake",
+                NormalizedUserName = "FAKE",
+                Email = "fake@user.com"
+            };
+            await context.Set<ApplicationUser>().AddAsync(testUser);
+
+            await context.Set<ApplicationUser>().AddAsync(testUser2);
```

Comment:
I guess you can use the method `SeedEntities`  for seeding multiple entities that Andrii created (in the `TestBase` class). Same as here:
![image](https://user-images.githubusercontent.com/56138100/140653351-95bec0be-4988-412d-bcbf-880dfa22f3df.png)

Reply:
I forgot to merge the master to my branch so I didn't have the SeedEntities code when I made the tests I'll make the changes to use the SeedEntities now